### ISSUE TITLE
Disable maxRecentUploads / trimRevisions

### ISSUE THREAD

**Issue by buschtoens** [mwpastore/ember-cli-deploy-sql#5]
I find that with rapid release cycles and multitenancy in mind, `maxRecentUploads` is a dangerous feature, that should not be enabled by default.

I'm okay with keeping it on by default (though I wouldn't), but it'd be super sweet, if you could disable it by passing something like `Infinity` or `false`.

Setting it to `Infinity` already works, but `trimRevisions` is nevertheless executed in vain.

I'm up for a PR. :slightly_smiling_face:

**Comment by mwpastore** [comment]
Yeah, this is just something I carried over as-is from ember-cli-deploy-redis. I agree that it's a weird default; I recall wanting to disable it or at least set it to a much higher value (like 100 instead of 10). I think it makes sense to set it to *something* by default to prevent the table from growing forever, but that may just be my old-school sysadmin brain fooling me. Feel free to submit a PR allowing users to disable it by setting it to `false`.
